lib/socialcast-git-extensions/cli.rb

```diff
@@ -128,13 +128,23 @@ def backportpr(pull_request_num, maintenance_branch)
         socialcast_reviewer = socialcast_track_reviewer('Backport')
 
         pull_request_data = github_api_request('GET', "repos/#{repo}/pulls/#{pull_request_num}")
-        commits_data = github_api_request('GET', pull_request_data['commits_url'])
 
-        non_merge_commits_data = commits_data.select { |commit_data| commit_data['parents'].length == 1 }
-        shas = non_merge_commits_data.map { |commit| commit['sha'] }
+        commits_to_cherry_pick = []
+        commit_count = pull_request_data['commits']
+        commits_checked_count = 0
+        page = 1
+
+        while commits_checked_count < commit_count
+          commits_data = github_api_request('GET', pull_request_data['commits_url'] + "?page=#{page}")
+          commits_checked_count += commits_data.size
```

Comment:
For safety you probably want to fail loudly here if commits_data.size is zero (vs spinning forever).